src/frontend/.prettierrc

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Again - does this file relate tot he UI or the tests?

Reply:
yes, there were a few formatting inconsistencies with the quotes, tabs, and alignments which prettier fixes